target/linux/ramips/dts/rt5350_tenda_a6.dts

```diff
@@ -0,0 +1,111 @@
+/dts-v1/;
```

Comment:
Please add an SPDX license identifier.

Reply:
Hrm, where do I find that?

Reply:
In all recently (at least for the last year) added DTS files.

Reply:
https://spdx.org/licenses/